### ISSUE TITLE
Option to add a border around images?

### ISSUE THREAD

**Issue by tobyhodges** [swcarpentry/styles#513]
In the process of developing a lesson using the remote theme, we ran into some trouble with screenshots embedded in the body of an episode: we had a short paragraph sandwiched between two screenshots, both of which showed blank/dark grey text on a white background (they are screenshots of the GitHub interface). One developer noticed that it was very easy to miss that paragraph between the two screenshots and that got us to wondering how we might more clearly mark out figures from text in the lesson.

Would it be possible to add a mechanism to place a border/shadow around images displayed on lesson pages? This might not be desirable for every lesson, or indeed even every image within a given lesson, but would be very helpful in cases like ours. Given the topics covered in Carpentries lessons, I'm sure we're not the only people using screenshots in our lesson, so I suspect this would help others too.

**Comment by fmichonneau** [comment]
maybe we could create a new CSS class that would take care of this?

**Comment by maxim-belkin** [comment]
The solution is to add CSS specification right after the image in question, e.g.:

```
![Alt text](image.svg){: style="border: solid 1px red; background: #eee;"}
```
It might be possible to add class specification as well (such as `{: .image-with-borders}`) and then define that class in `assets/css/lesson.scss` but I haven't tested this yet.

**Comment by fmichonneau** [comment]
Toby implemented this in #514
